addons/website_event/controllers/main.py

```diff
@@ -257,8 +259,23 @@ def registration_confirm(self, event, **post):
             registration['event_id'] = event
             Attendees += Attendees.sudo().create(
                 Attendees._prepare_attendee_values(registration))
-
+        urls = event._get_urls(Attendees.ids)
         return request.render("website_event.registration_complete", {
             'attendees': Attendees,
             'event': event,
+            'google_url': urls.get('google_url'),
+            'iCal_url': urls.get('iCal_url')
         })
+
+    @http.route(['/event/<model("event.event"):event>/ics/<string:name>.ics'], type='http', auth="public", website=True)
+    def calendar_appointment_ics(self, event, **kwargs):
```

Comment:
What is the purpose of name ? Why don't you define attendees from kwargs ? What happen if it is not set ?

+ update method and route naming